events/listener.go

```diff
@@ -101,7 +101,7 @@ func (router *EventRouter) Start(ready chan<- bool) (err error) {
 		case worker := <-workers:
 			go worker.DoWork(line, handlers, router.apiClient, workers)
 		default:
-			log.Printf("No workers available dropping event.")
+			log.Debug("No workers available dropping event.")
```

Comment:
I'd like this to be Info level. If we consistently don't have workers available, we need to know it so that we can tune the number of workers.